src/CSI.IBTA.UserService/Mapping/MapperProfile.cs

```diff
@@ -17,8 +17,8 @@ public MapperProfile()
                     user.Lastname,
                     user.AccountId,
                     user.Employer != null ? user.Employer.Id : null,
-                    user.Emails[0].EmailAddress,
-                    user.Phones[0].PhoneNumber)
+                    user.Emails != null && user.Emails.Count > 0 ? user.Emails[0].EmailAddress : null,
+                    user.Phones != null && user.Phones.Count > 0 ? user.Phones[0].PhoneNumber : null)
```

Comment:
EmailAdress and PhoneNumber are not nullable for UserDto, so if employer does not have one, we should assign an empty string. 
```suggestion
                    user.Phones != null && user.Phones.Count > 0 ? user.Phones[0].PhoneNumber : "")
```

This prevents us from dealing with null strings, as attempting to perform operations on a null string (such as accessing its properties or calling methods) can lead to a `NullReferenceException`